web/massastation/src/pages/store/MyPlugin.tsx

```diff
@@ -0,0 +1,90 @@
+import { useState } from 'react';
+
+import {
+  Button,
+  Certificate,
+  MassaWallet,
+  Plugin,
+} from '@massalabs/react-ui-kit';
+import { FiArrowUpRight, FiRefreshCcw, FiTrash2 } from 'react-icons/fi';
+import { IMassaPlugin } from './MyStation';
+
+enum PluginStatus {
+  Up = 'Up',
+  Down = 'Down',
+}
+
+function MyPlugin({ plugin }: { plugin: IMassaPlugin }) {
+  const [myPlugin, setMyPlugin] = useState<IMassaPlugin>(plugin);
+  function handlePluginState(method: string) {
+    // TODO :
+    // const {mutate } = usePost<any>("plugin-manager");
+    // const ok = mutate({method:"${method}"})
+    const newStatus = method === 'start' ? PluginStatus.Up : PluginStatus.Down;
+    const updatedPlugin = { ...myPlugin, status: newStatus };
+    setMyPlugin(updatedPlugin);
+  }
+  const argsOn = {
+    preIcon:
+      myPlugin.author === 'MassaLabs' ? (
+        <MassaWallet variant="rounded" />
+      ) : (
+        <img src={myPlugin.logo} />
+      ),
+    topAction: (
+      <Button onClick={() => handlePluginState('stop')} variant="toggle">
+        on
+      </Button>
+    ),
+    title: `${myPlugin.name} `,
+    subtitle: `${myPlugin.author}`,
+    subtitleIcon: myPlugin.author === 'MassaLabs' ? <Certificate /> : <></>,
+    content: [
+      // conditionally render the update icon based on the updatable property
+      myPlugin.updatable && (
+        <Button variant="icon" onClick={() => console.log('reload')}>
+          <FiRefreshCcw className="text-s-warning" />
+        </Button>
+      ),
+      <Button variant="icon" onClick={() => console.log('arrow')}>
+        <FiArrowUpRight />
+      </Button>,
+      <Button variant="icon" onClick={() => console.log('trash')}>
+        <FiTrash2 />
+      </Button>,
+    ],
+  };
+
+  const argsOff = {
+    preIcon:
+      myPlugin.author === 'MassaLabs' ? (
```

Comment:
Wow!  Really!? 
`author === 'MassaLabs'`🚫 
If they decide to put `Massa Labs` `ML` then we are die!

Reply:
I will check with marketing team, than create a variable as unique source of truth, what do you think @mazmassa

Reply:
SO i created an Array that will store all MassaLabs nomination adn a fn that is called in MyPlugin, that returns true if the author is equal to a nomination in the array

Reply:
Yes! Great! Thanks @pivilartisant . We can use like that: 

https://github.com/massalabs/thyra/pull/760#discussion_r1230779088